core/src/main/java/eu/fasten/core/maven/resolution/MavenDependentsResolver.java

```diff
@@ -49,7 +48,7 @@ public Set<ResolvedRevision> resolve(String gav, ResolverConfig config) {
         }
 
         var dependents = new HashSet<ResolvedRevision>();
-        resolve(pom, config, dependents, new HashSet<>(), false, COMPILE);
+        resolve(pom, config, dependents, new HashSet<>(), false, COMPILE, 1);
```

Comment:
What is this hard-coded number, `1` ? It's depth, I guess. Maybe it would be better to introduce a constant, like `DIRECT`

Reply:
This is the depth, yes. You can see this in the name of the method parameter. Given that the depth is constantly increased in the recursive calls, how would I use a constant here? On a side note, I would like to replace the recursive solution with a work queue anyways (which eliminates the need for this parameter), but did not find the time to do this in this refactoring...